content/docs/concepts/asm/_index.md

```diff
@@ -0,0 +1,276 @@
+
+- Collateralization:
+  - A CDP must maintain adequate collateral value at all times. A _minimum collateralization ratio_ sets the minimum ACU value of collateral required to _borrow_ 1 ACU value of debt. This ratio must be strictly greater than `1.0`.
+  - Collateral must be an ERC 20 token. The accepted collateral token is the protocol asset Newton.
+
+- Liquidation ratio:
+  - A CDP must maintain adequate collateral value at all times. A _liquidation ratio_ sets the minimum ACU value of collateral required to _maintain_ 1 ACU value of debt. 
+  - The liquidation ratio must be strictly less than the _minimum collateralization ratio_.
+
+- Liquidation Condition:
+  - A CDP is _liquidatable_ when the CDP's _collateralisation ratio_ falls below the _liquidation ratio_:
+  - If the _liquidation condition_ is met, the only permitted operations on the CDP are:
+    - For the _Borrower_, the CDP Owner, to pay back Auton until and repay the CDP or bring the position back within the _liquidation ratio_
+    - For a _Liquidator_ to liquidate the position.
+
+
+### CDP lifecycle
+
+The sequence of lifecycle events for a CDP is:
+
+- CDP is opened.
+  - Borrower determines their borrowing and collateral requirements. To do this, the borrower can call Stabilization Contract functions, see [`collateralPrice()`](/reference/api/asm/stabilization/#collateralprice), [`minimumCollateral()`](/reference/api/asm/stabilization/#minimumcollateral).
+  - Borrower opts to open a CDP, becoming a CDP Owner. The CDP Owner then approves the Stabilization Contract to spend Collateral Token (NTN) on their behalf for the amount of collateral to be deposited:
+    - Calls the Collateral Token Contract (i.e. Autonity Protocol Contract) to [`approve()`](/reference/api/aut/#approve) the [Stabilization Contract address](/concepts/architecture/#protocol-contract-account-addresses) as a spender on their behalf and set the [`allowance()`](/reference/api/aut/#allowance) of collateral token that the Stabilization Contract can spend.
+    - Calls the Stabilization Contract to deposit Collateral Token to a CDP, calling the contract's [`deposit()`(/reference/api/asm/stabilization/#deposit) to deposit collateral.
+  - Stabilization Contract creates the CDP, using the ERC20 allowance mechanism to execute the collateral deposit. A CDP object is created, and the [CDP attributes](/concepts/asm/#stabilization) are populated.
+
+- CDP is maintained and serviced. The CDP Owner maintains the CDP, opting to increase or decrease borrowing and deposited collateral within [CDP primitive constraints](/concepts/asm/#cdp). The CDP Owner may:
+
+  - _Borrow_: CDP Owner borrows Auton from CDP against collateral. Constraint checks are applied:
+    - the amount borrowed must not exceed the _borrow limit_ for the deposited collateral value
+    - the _principal_ debt must meet the _minimum debt requirement_
+    - the borrowing does not make the CDP meet the _liquidation condition_.
```

Comment:
- the borrowing does not **~make~ decrease** the CDP **~meet the _liquidation condition_~ collateralization ratio below the _minimum collateralization ratio_**.

Reply:
IMO borrowing shall not decrease the CDP's collateralization ratio < minimum collateralization ratio.